docker/.env

```
@@ -542,5 +542,6 @@ SMQ_GRAFANA_ADMIN_PASSWORD=supermq
 ## Allow unverified user to access
 SMQ_ALLOW_UNVERIFIED_USER=true
 
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)